RELEASE.md

```diff
@@ -1,33 +1,54 @@
-## Making a release of the Checker Framework Gradle plugin
+## Making and merging pull requests
 
-This document describes how to make a release of the Checker Framework
-Gradle plugin, which this repository contains.
+The Checker Framework Gradle plugin (which this repository contains)
+uses a continuous-delivery style of releases: any time that the
+behavior of the plugin changes, a new release is cut.
 
-### Prequisites
+### Making a pull request
+
+All pull requests should be made from a feature branch. Never push
+directly to `master`.
+
+When you make a pull request that will change any behavior
+of the plugin (including bug fixes and new features), update the version string on 
```

Comment:
These 4 steps are not about "update the version string", so this text is confusing.